lib/datadog/appsec/security_engine/runner.rb

```diff
@@ -9,10 +9,14 @@ module SecurityEngine
       class Runner
         SUCCESSFUL_EXECUTION_CODES = [:ok, :match].freeze
 
-        def initialize(handle_ref)
+        attr_reader :ruleset_version, :waf_addresses
+
+        def initialize(handle_ref:, ruleset_version:, waf_addresses:)
```

Comment:
If addresses could be retrieved via `waf_handle`, why don't we do it like in engine?

```
@waf_addresses = waf_handle.known_addresses
```

The problem of converting normal arguments to the kwagrs is that we will loose the sight of what is the main argument and what is an additional (potentially optional).

As we could extract addresses from handle, we just need to know the version of the ruleset used and it's an additional argument. So in this case we can reduce interface to


```suggestion
        def initialize(handle_ref, ruleset_version:)
```